app.js

```diff
@@ -1,5 +1,20 @@
 const express = require('express');
-
+const { getCategories } = require('./controller') 
 const app = express();
 
+app.get('/api/categories', getCategories)
+
+
+app.use( (error, request, response, next) => {
```

Comment:
Well done for handling this in the app. It's easy to mistakenly put this in the model. Good work.